www/perl-lib/SMT/Rest/SCCAPIv2.pm

```diff
@@ -638,7 +646,8 @@ sub _registrationResult
     my $p = $self->_getProduct($product_id);
     if ( scalar(keys %{$p}) == 0)
     {
-        $self->request()->log_error("Incomplete Registration found. Unable to find the product");
+        my $guid = $self->user();
+        $self->request()->log_error("Incomplete Registration found. Unable to find the product $product_id, client ID: $guid");
```

Comment:
why not here also use `sprintf`?

Reply:
Other string contain method calls and hash elements. Here it's just two scalars. Why over-complicate things?

Reply:
Just thought the whole source code seems to use the `sprintf` thing. but ok sure its over-complicating things

Reply:
but anyway: